tools/topology/topology2/production/tplg-targets-ace2.cmake

```diff
@@ -27,6 +27,10 @@ SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,SDW_DMIC_STREAM=Capture-SmartMic,\
 SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 
+"cavs-sdw\;sof-lnl-rt713-l0-rt1318-l1\;PLATFORM=lnl,NUM_SDW_AMP_LINKS=1,\
+SDW_SPK_STREAM=SDW1-Playback,SDW_AMP_FEEDBACK=false,\
+HDMI1_ID=6,HDMI2_ID=7,HDMI3_ID=8"
```

Comment:
The HDMI IDs are probably wrong. It is expected to be 5, 6, and 7 which are the default values if there is no PCH DMIC.